### PR TITLE
doc: document that pre-/post-processing is not supported with multi-model

### PR DESCRIPTION
**Pull request by laurenyu**
*Issue #, if available:*
#107

*Description of changes:*
There is currently no support for pre-/post-processing with multi-model containers, but our documentation doesn't say that.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -426,7 +428,7 @@ For those using Elastic Inference set the image following this format instead:
 ```
 
 Where `REGION` is your AWS region, such as "us-east-1" or "eu-west-1"; `SAGEMAKER_TENSORFLOW_SERVING_VERSION`, `SAGEMAKER_TENSORFLOW_SERVING_EIA_VERSION`, `TENSORFLOW_INFERENCE_VERSION`, `TENSORFLOW_INFERENCE_EIA_VERSION` are one of the supported versions mentioned above; and "gpu" for use on GPU-based instance types like ml.p3.2xlarge, or "cpu" for use on CPU-based instances like `ml.c5.xlarge`.
- 
+
```
Comment (aaronmarkham) [inline README.md]: Formatting is getting messed up down here. I think if you put the directory structure stuff in a code block, you'll avoid this.

Comment (laurenyu) [reply README.md]: what part are you referring to? the only directory structure stuff I see is already in a code block:

Screen Shot 2020-01-17 at 1 14 02 PM

Comment (aaronmarkham) [reply README.md]: Maybe it's something else then... but you can see above how everything is bolded... that's usually when some code block gets messed up or special character is triggering the markdown to start rendering like that.

Comment (laurenyu) [reply README.md]: ah. upon further inspection, I think it's the use of double underscores in the code directory stuff. It looks okay in the rendering, though.

**Pull request merged**